docs/procedures.md

```diff
@@ -129,4 +119,12 @@ What are the changes:
 
 ## Deployment procedure using Ansible
 
-Moved to relevant repository `README.md`.
+1. Back up the database through the AWS console (RDS > Databases > prod-amy > Actions (top right) > Take snapshot).
```

Comment:
I'm not sure how I feel about publishing our production cluster name. Maybe a more generic instruction would be better, e.g. `RDS -> Databases -> AMY database cluster -> Actions (top right) -> Take snapshot`?
This is also more futureproof (cluster name can change).

Reply:
We should have a naming scheme, too. As you saw, naming the snapshot is not easly.

What about "v4.2.0_YYYYMMDD" scheme? Do you remember if it allowed dots and underscores in the snapshot name?

Reply:
From the AWS snapshot field hint: `Snapshot identifier is case insensitive, but stored as all lower-case, as in "mysnapshot". Cannot be null, empty, or blank. Must contain from 1 to 255 alphanumeric characters or hyphens. First character must be a letter. Cannot end with a hyphen or contain two consecutive hyphens.`